src/dgraph.ts

```diff
@@ -32,11 +32,11 @@ async function dqlQuery(query: string, variables: Record<string, any> = {}): Pro
   return response.json();
 }
 
-async function dqlMutate(mutate: string): Promise<GraphQLResponse> {
+async function dqlMutate(mutate: string | any): Promise<GraphQLResponse> {
   const response = await fetch(`${process.env.DGRAPH_URL}/mutate?commitNow=true`, {
     method: "POST",
     headers: {
-      "Content-Type": "application/rdf",
+      "Content-Type": typeof mutate === 'object' ? "application/json" : "application/rdf",
       "X-Auth-Token": process.env.DGRAPH_TOKEN || ""
     },
     body: mutate
```

Comment:
```suggestion
    body: typeof mutate === 'object' ? JSON.stringify(mutate) : mutate
```

It worked for me after this change. I had the following lambda function:
```
async function newAuthor({args, dql}) {
    // lets give every new author a reputation of 3 by default
    const mutation = {
        set: [
            {
                uid: "_:newAuth",
                "Author.name": args.name,
                "Author.reputation": 3.0,
                "dgraph.type" : "Author"
            }
        ]
    };
    const res = await dql.mutate(mutation);
    return res.data.uids.newAuth
}
```